script/foundry/utils/DeployHelper.sol

```diff
@@ -157,8 +158,16 @@ contract DeployHelper is Script, BroadcastManager, JsonDeploymentHandler, Storag
         _endBroadcast(); // BroadcastManager.s.sol
     }
 
+    function create3Deploy(bytes32 salt, bytes calldata creationCode, uint256 value) external returns (address) {
+        return CREATE3.deploy(salt, creationCode, value);
+    }
+
     function _deployProtocolContracts() private {
         require(address(erc20) != address(0), "Deploy: Asset Not Set");
+        bytes32 ipAccountImplSalt = keccak256(
+            abi.encode(type(IPAccountImpl).creationCode, address(this), block.timestamp)
```

Comment:
the block.timestamp is still random?  do we want to control the salt during the deployment?

Reply:
block.timestamp is for making sure the salt is unique; do not have to be random.